src/pocketmine/Player.php

```diff
@@ -2980,6 +2980,14 @@ public function sendTip($message){
 		$pk->message = $message;
 		$this->dataPacket($pk);
 	}
+	
+	public function whisper($from, $message){
+		$pk = new TextPacket();
+		$pk->type = TextPacket::TYPE_WHISPER;
+		$pk->source = $from instanceof Player ? $from->getDisplayName() : $from;
```

Comment:
Are you sure this is a good idea? What if the parameter is some other non-player object? I think it would be a better idea to restrict this directly to strings only. Please use type-hinting where possible.